troubleshooting.md

```diff
@@ -30,11 +30,28 @@ $ oc adm must-gather --image=registry.redhat.io/rhcam-1-2/openshift-migration-mu
 
 ## Scale a quiesced application back
 
-TBD
+If the migrated source application was quiesced during your migration,
+you will also need to scale it back to its initial replica count. This can be
+done manually by editing the deployment primitive (Deployment, DeploymentConfig, etc.)
+and setting the `spec.replicas` field back to the original, non-zero value:
+
+```
+$ oc edit deployment <deployment-name>
+```
```

Comment:
Adding an example of using `oc scale --replicas=` would be good.

Assuming we track the original number of replicas in an annotation on source this would be good to call out, so folks can look at the existing annotations to get the exact replicas used prior.